.gitattributes

```diff
@@ -3,5 +3,10 @@
 
 /.gitattributes export-ignore
 /.gitignore export-ignore
+.travis.yml
```

Comment:
does this need a leading `/`?

Reply:
```suggestion
/.travis.yml
```

I honestly don't know if they need them but I keep it there to be safe, will add.

Reply:
yeah unsure as well 🤷‍♂